String/CheckPangram.js

```diff
@@ -0,0 +1,17 @@
+// Pangram is a uniques sentence that contains every letter of the alphabet
+// The alphabet has to appear at least once
+// Pangram Check is case sensitve
+function checkPangram (sentence) {
+  // Create an array of 26 characters
+  const alphabets = 'abcdefghijklmnopqrstuvwxyz'

```

Comment:
```suggestion
  const ALPHABETS = 'abcdefghijklmnopqrstuvwxyz'
```
Move it out of the function body

Reply:
noted